modules/webgl/src/adapter/resources/webgl-vertex-array.ts

```diff
@@ -54,14 +54,14 @@ export class WEBGLVertexArray extends VertexArray {
   }
 
   /**
-  // Set (bind) an elements buffer, for indexed rendering.
-  // Must be a Buffer bound to GL.ELEMENT_ARRAY_BUFFER. Constants not supported
+  // Set (bind/unbind) an elements buffer, for indexed rendering.
+  // Must be a Buffer bound to GL.ELEMENT_ARRAY_BUFFER or null. Constants not supported
    * 
    * @param elementBuffer 
    */
   setIndexBuffer(indexBuffer: Buffer | null): void {
     const buffer = indexBuffer as WEBGLBuffer;
-    if (buffer?.glTarget !== GL.ELEMENT_ARRAY_BUFFER) {
+    if (buffer && buffer.glTarget !== GL.ELEMENT_ARRAY_BUFFER) {
```

Comment:
Ahh, subtle, I can see myself routinely collapsing this in an attempt to be more succinct and introducing this bug...

Maybe add a comment line or make it into nested if statements?

Reply:
Also a test case would be good but might be hard to put together.